sdk/cosmos/azure-cosmos-benchmark/src/main/java/com/azure/cosmos/benchmark/Main.java

```diff
@@ -37,15 +43,18 @@ public static void main(String[] args) throws Exception {
             if (cfg.isSync()) {
                 syncBenchmark(cfg);
             } else {
-                if(cfg.getOperationType().equals(ReadThroughputWithMultipleClients)) {
+                if (cfg.getOperationType().equals(ReadThroughputWithMultipleClients)) {
                     asyncMultiClientBenchmark(cfg);
-                } else if(cfg.getOperationType().equals(CtlWorkload)) {
+                } else if (cfg.getOperationType().equals(CtlWorkload)) {
                     asyncCtlWorkload(cfg);
                 } else if (cfg.getOperationType().equals(LinkedInCtlWorkload)) {
                     linkedInCtlWorkload(cfg);
-                }
-                else {
-                    asyncBenchmark(cfg);
+                } else {
+                    if (cfg.isEncryptionEnabled()) {
```

Comment:
can we follow the same pattern as the other else if here?

```java
else if (cfg.isEncryptionEnabled()) 
```

instead of 
```java
else {
   if (cfg.isEncryptionEnabled()) 
}
```

Reply:
done